src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -324,7 +384,12 @@ public class AutocompleteResponse
             public string Index { get; set; }
             public long TotalHits { get; set; }
             public DateTimeOffset LastReopen { get; set; }
+        }
 
+        public class SearchService
```

Comment:
Yes, move to its own file and make immutable. That it, make the properties `{ get; }` and constructor accept all property values.